### PR TITLE
REPLACE String Function: #13

### PR DESCRIPTION
**Pull request by Lilit0x**
Replace string function.

### CODE REVIEW

File: crates/gitql-ast/src/function.rs [hunk 1]
```diff
@@ -242,6 +250,26 @@ fn text_char(inputs: Vec<Value>) -> Value {
     return Value::Text("".to_string());
 }
 
+fn text_replace(inputs: Vec<Value>) -> Value {
```
Comment (AmrDeveloper) [inline crates/gitql-ast/src/function.rs:253]: Why not using the std replace function from string type

```rs
let text = inputs[0].as_text();
let old_string = &inputs[1].as_text();
let new_string = &inputs[2].as_text();
return Value::Text(text.replace(old_string, new_string));
```

**Review by AmrDeveloper — APPROVED** [review]

File: crates/gitql-ast/src/function.rs (hunk 1 above)
Comment (AmrDeveloper) [inline crates/gitql-ast/src/function.rs:253]: Why not using the std replace function from string type

```rs
let text = inputs[0].as_text();
let old_string = &inputs[1].as_text();
let new_string = &inputs[2].as_text();
return Value::Text(text.replace(old_string, new_string));
```

**Review by AmrDeveloper — CHANGES_REQUESTED** [review]

File: crates/gitql-ast/src/function.rs (hunk 1 above)
Comment (Lilit0x) [reply crates/gitql-ast/src/function.rs:253]: The search has to be case insensitive, but `.replace` is std is case sensitive. If I use `.replace` this case won't pass.
`SELECT REPLACE("ABC ABC ABC", "a", "c") as replacedText`

Comment (AmrDeveloper) [reply crates/gitql-ast/src/function.rs:253]: Ok yes i will search if we can make replace ignore case, and please resolve the conflicts

**Review by AmrDeveloper — APPROVED** [review]

**Comment by AmrDeveloper** [comment]
Thank you @Lilit0x

**Pull request merged**